bot/cogs/moderation/management.py

```diff
@@ -135,10 +135,10 @@ def infractions_cog(self) -> Infractions:
         user = ctx.guild.get_member(user_id)
 
         if user:
-            user_text = f"{user.mention} (`{user.id}`)"
+            user_text = f"{user.mention} (`{user}`)"
```

Comment:
As stated in the issue, the nickname should be in parenthesis. Right now, this will show the username + discriminator. Keep in mind the nick may be `None`. In that case, the parenthesised info can also be omitted.

Reply:
Actually, in case the nick is none showing the username would be good.